### PR TITLE
rename target/decoy column in MzTabExporter

### PR DESCRIPTION
**Pull request by lars20070**
Replaces the `opt_global_target_decoy` column in `MzTabExporter` by the `opt_global_cv_MS:1002217_decoy_peptide` column by remapping like:
```
target --> 0
decoy --> 1
target+decoy --> 0
```
Addresses #3943. Follows the fixes in PR #3973.

### CODE REVIEW

**Comment by timosachsenberg** [comment]
Why target+decoy to 0?

**Comment by lars20070** [comment]
@jpfeuffer Can you see why jenkins is not finishing?

File: src/openms/source/FORMAT/MzTab.cpp [hunk 1]
```diff
@@ -587,6 +587,29 @@ namespace OpenMS
     mz_tab_version.fromCellString(String("1.0.0"));
   }
 
+	// static method remapping the target/decoy column from an opt_ to a standardized column
+	static void remapTargetDecoy_(std::vector<MzTabOptionalColumnEntry> &opt_entries)
+	{
```
Comment (lars20070) [inline src/openms/source/FORMAT/MzTab.cpp]: [Travis](http://cdash.openms.de/testDetails.php?test=56758007&build=311424) code style complaint [here](). Cannot see where the space is missing.
```
/home/travis/build/OpenMS/OpenMS/src/openms/source/FORMAT/MzTab.cpp:592:  Missing space before {  [whitespace/braces] [5]
```

**Comment by lars20070** [comment]
rebuild jenkins

File: src/openms/source/FORMAT/MzTab.cpp (hunk 1 above)
Comment (cbielow) [reply src/openms/source/FORMAT/MzTab.cpp]: should be `std::vector<MzTabOptionalColumnEntry>& opt_entries)`?

**Comment by jpfeuffer** [comment]
rebuild jenkins

**Comment by lars20070** [comment]
[Test output](http://cdash.openms.de/testDetails.php?test=56783048&build=311550) does not clarify why Windows fails.

**Comment by jpfeuffer** [comment]
Exit code -1073740791 = STATUS_STACK_BUFFER_OVERRUN.

**Comment by jpfeuffer** [comment]
See #3973 . Maybe run it on your computer with AddressSanitizer and/or valgrind. Not sure why linux does not complain but I would think it is not a coincidence.

**Comment by jpfeuffer** [comment]
rebuild jenkins

**Comment by lars20070** [comment]
@jpfeuffer Thanks for the fix.
There are still *memory leaks* in Python.

**Pull request merged**